.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @citizensadvice/software-engineering
```

Comment:
This team doesn't exist yet but suggesting that we create it for these shared repositories.